### PR TITLE
Fixes for #1

### PR DESCRIPTION
**Pull request by ExposedCat**

### CODE REVIEW

File: src/services/user.ts [hunk 1]
```diff
@@ -4,19 +4,19 @@ export function buildName(firstName: string, lastName?: string) {
...
 	name: string
 }): Promise<User> {
-	const playerObject = {
+	const userObject = {
 		userId: args.userId,
 		name: args.name
 	} as User
 
-	await args.db.user.insertOne(playerObject)
+	await args.db.user.insertOne(userObject)
```
Comment (s373r) [inline src/services/user.ts:17]: probably `*Object` part is extra

analogy: then we create an array of users we don't name it `usersArray` we name `users`

here the same thing: do not add type into name (it isn't context in this case)

File: src/services/user.ts [hunk 2]
```diff
@@ -4,19 +4,19 @@ export function buildName(firstName: string, lastName?: string) {
...
 
-	return playerObject
+	return userObject
 }
 
 export async function getOrCreatePlayer(args: {
```
Comment (s373r) [inline src/services/user.ts:22]: getOrCreatePlayer() -> getOrCreateUser()

File: src/config/bot.ts [hunk 3]
```diff
@@ -60,6 +61,9 @@ export async function startBot(database: Database) {
 	extendContext(bot, database)
 	setupMiddlewares(bot, i18n)
 	setupControllers(bot)
+
+	// NOTE: Resolves only when bot is stopped
+	// so give it a second to start instead of `await`
```
Comment (s373r) [inline src/config/bot.ts:66]: [not for fix comment]: that is a good example of great comment

File: src/services/user.ts (hunk 1 above)
Comment (ExposedCat) [reply src/services/user.ts:17]: @s373r `Object` here explicitly highlights that it's a plain object - not a class/db entry

**Pull request merged**